.github/workflows/node-versions.yml

```diff
@@ -1,6 +1,6 @@
 # This special file aims to run node tests for each relevant node version.
 # A relevant node version can be: active, current, maintenance or EOL with some months of tolerance
-# For more details, please check ./scripts/node-versions.js
+# For more details, please check https://github.com/evertonfraga/testable-node-versions
```

Comment:
This left me a bit confused, then I followed the path and realized that the `npx testable-node-versions` command within this file is indeed installing the package from the above-changed-location, being a script from Everton (former EthereumJS team member).

I would have a strong tendency to internalize this script (not part of this PR), I am not feeling well with these kind of super-implicit hidden external dependencies.

Reply:
I'm good with that.  It was internal originally and then I guess Ev moved it out to make it usable wherever else.  Can put that as a new issue to resolve.